js/tc39/README.md

```diff
@@ -7,15 +7,23 @@ Ways to use it:
    that was tested with this module
 2. Run `go test &> out.log`
 
-The full list of failing tests, and the error, is in `breaking_test_errors-*.json`. All errors list there with the corresponding error will *not* be counted as errors - this is what the test expects, those specific errors.
-Due to changes to Sobek it's common for the error to change, or there to be now a new error on the previously passing test, or (hopefully) a test that was not passing but now is.
-In all of those cases `breaking_test_errors-*.json` needs to be updated. Run the test with `-update` flag to update: `go test -update`
-
-NOTE: some text editors/IDEs will try to parse files ending in `json` as JSON, which given the size of `breaking_test_errors-*.json` might be a problem when it's not actually a JSON (before the edit). So it might be a better idea to name it something different if editing by hand and fix it later.
+The full list of failing tests, and the error, is in `breaking_test_errors-*.json`. All errors list there with the corresponding error will *not* be counted as errors - this is what the test expects, those specific errors. See reasons for this at the end of this document.
 
 This is a modified version of [the code in the original goja
 repo](https://github.com/dop251/goja/blob/master/tc39_test.go) that Sobek was forked from.
 
+## Maintaining the tests
+
+There are few things to keep in mind when maintaining the tests.
+
+* For the most of the version we aim to have the same version of the test suite as the one that [uses Sobek](https://github.com/grafana/sobek/blob/main/.tc39_test262_checkout.sh#L3). So if the Sobek brings a new version of the test suite, we should update the version of the test suite in this package as well.
```

Comment:
The beginning of the sentence is a little confusing, I think

Reply:
Right, was trying to write this down in-between meetings :sweat_smile: 

```suggestion
* For the most time, we aim to have the same version (commit hash) of the test suite as the one that [uses Sobek](https://github.com/grafana/sobek/blob/main/.tc39_test262_checkout.sh#L3). So if Sobek brings a new version of the test suite, we should update the version of the test suite in this package as well.
```

Reply:
@joanlopez, how about this version?

Reply:
Yeah, now looks better. Thanks! 🙇🏻